tools/cal_scripts/ctd_cal_parser.py

```diff
@@ -151,6 +152,8 @@ def main():
     lookup = get_uid_serial_mapping('CTD/ctd_lookup.csv')
     for path, directories, files in os.walk('CTD/manufacturer'):
         for file in files:
+            if file == '.DS_Store':
```

Comment:
if you're skipping this hidden file, might as well skip all hidden files: `if file[0] == '.':`

Reply:
See latest commit